src/core/tools/common/simple_logger.h

```diff
@@ -26,11 +26,9 @@
 
 /*
  * Description:
```

Comment:
`Description` section should be removed completly, and move the class description closer to the class definition.